app/views/applications/show.html.erb

```diff
@@ -0,0 +1,54 @@
+<style>
+  #reason {
+    width: 300px;
+    height: 100px; 
+  }
+</style>
+
+<%= render partial: 'application/header', layout: 'header' %>
+<%= render partial: 'application/body', layout: 'body' %>
+
+<div class="myDiv" >
+  <h1>Name on Application: <%= @application.name %></h1>
+</div>
+
+<div class="body" >
+<p><u>Address</u>: <%= @application.full_address %> </p>
+<p><u>Why would you make a good home?</u>: <%= @application.description %> </p>
+<p><u>Status</u>: <%= @application.status %> </p>
+
+<p><u>Pets I'd Like to Adopt</u>:</p>
+<% @pets.each do |pet| %>
+  <%= link_to "#{pet.name}", "/pets/#{pet.id}", method: :get %>
+<%end%>
+
+<% if @application.status == "In Progress" %>
+  <h2><u>Add a pet to this application</u>:</h2>
+  <%= form_with url:"/applications/#{@application.id}", method: :get, local: :true do |f| %>
```

Comment:
Sometimes Turbo gets mad when we create forms that redirect back to the same controller action -- if you saw any weirdness with that, make sure you include `data: { turbo: false }`